internal/service/advancedclustertpf/resource_compatibility_reuse.go

```diff
@@ -3,14 +3,28 @@ package advancedclustertpf
 import (
 	"context"
 	"fmt"
+	"strconv"
 	"strings"
 
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/constant"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
 	"github.com/spf13/cast"
 	admin20240530 "go.mongodb.org/atlas-sdk/v20240530005/admin"
 	"go.mongodb.org/atlas-sdk/v20241113002/admin"
 )
 
+func IsMajorVersionHigherGreaterOrEqual(input *string, version float64) *bool {
```

Comment:
`IsMajorVersionGreaterOrEqual` ?

Reply:
Ooops 🤣 thank you!

Reply:
8a6c7940501887150b2062935bad580bb89accda